shiny/ui/_chat.py

```diff
@@ -148,8 +148,9 @@ def __init__(
         tokenizer: TokenEncoding | MISSING_TYPE | None = MISSING,
     ):
 
+        id = resolve_id(id)
         self.id = id
```

Comment:
I'm inclined to require that `id` is asserted of type string. Otherwise `self.user_input_id` will turn into `None_user_input`

```suggestion
        assert(isinstance(id, str))
        self.id = resolve_id(id)
```